src/token/components/TokenChip.tsx

```diff
@@ -0,0 +1,38 @@
+import { CSSProperties } from 'react';
+import { TokenChipReact } from '../types';
+
+const styles = {
```

Comment:
I like a lot how you organized the style.

@abcrane123, let's do something similar for your component as well.